ddtrace/tracer/tracer.go

```diff
@@ -295,8 +302,11 @@ func (t *tracer) worker(tick <-chan time.Time) {
 			// before the final flush to ensure no traces are lost (see #526)
 			for {
 				select {
-				case trace := <-t.out:
-					t.traceWriter.add(trace)
+				case info := <-t.out:
```

Comment:
```suggestion
				case trace := <-t.out:
```
Then, `trace.decision` and `trace.spans` makes complete sense.